addons/point_of_sale/static/src/css/pos.css

```diff
@@ -1548,9 +1548,14 @@ td {
     border-color: black;
     color: white;
 }
+.payment-screen .payment-buttons .button.highlight {
+    background: #e2e2e2 !important;
+    border-color: rgb(200,200,200) !important;
+    color: #555555 !important;
+}
 .payment-screen .payment-buttons .button.highlight .fa {
-    border-color: rgba(0, 0, 0, 0.109804);
-    background: rgba(0, 0, 0, 0.0980392);
+    background: #6EC89B !important;
+    border: solid 1px #64AF8A !important;
```

Comment:
try to remove !important and move this section (`.payment-screen .payment-buttons .button.highlight .fa`) after the next section (`.payment-screen .payment-buttons .button .fa`)